projects.html

```diff
@@ -49,9 +49,55 @@
         </div>
       </div>
     </nav>
-    <!-- Navigation bar complete -->>
+    <!-- Navigation bar complete -->
+     <!-- IMAGES -->
+ <!-- if u want to add more rows just add more table divisions -->
+	<div class="table"> <!-- ROW 1 -->
+    <div>
+     <img src="imgs/p1.png" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p2.png" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p3.jpg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p4.jpg" class="adjust">
+     </div>
+      </div>  
+   <div class="table"> <!-- ROW 2 -->
+    <div>
+     <img src="imgs/p5.jpeg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p6.jpg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p7.jpg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p8.jpg" class="adjust">
+     </div>
+      </div> 
+   <div class="table"> <!-- ROW 3 -->
+    <div>
+     <img src="imgs/p9.jpg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p10.jpg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p11.jpg" class="adjust">
+     </div>
+    <div>
+     <img src="imgs/p12.jpg" class="adjust">
```

Comment:
i can see this adjust class everywhere but couldnt find it in css file which has its properties. Can you let me know why is this used?

Reply:
my code of css is not there, i have Merging problem while trying of squash commit

Reply:
> my code of css is not there, i have Merging problem while trying of squash commit

Alright then the squashing right now and go for pushing the code including css. Check perfectly that your edited code of css html js is perfectly pushed in this PR.

Reply:
okk